tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -134,6 +134,9 @@ public TsFileSequenceReader(String file, boolean loadMetadataSize) throws IOExce
     }
     this.file = file;
     tsFileInput = FSFactoryProducer.getFileInputFactory().getTsFileInput(file);
```

Comment:
Current change is OK to resolve the issue. 
But to make it more safe, it's better to change the method `getTsFileInput(file)` to avoid making `null` as a possible returned value.
We can let the method `getTsFileInput` throw `IOException` if the file doesn't exist